src/main/java/com/emc/mongoose/storage/driver/pravega/io/DataItemSerializer.java

```diff
@@ -15,14 +15,15 @@
 				implements Serializer<I>, Serializable {
 
 	private final boolean useDirectMem;
-
+	private final boolean shouldRecordTime;
 	/**
 	 * @param useDirectMem Specifies whether should it use direct memory for the resulting buffer containing the
 	 *                     serialized data or not. Using the direct memory may lead to the better performance in case of
 	 *                     large data chunks but less safe.
 	 */
-	public DataItemSerializer(final boolean useDirectMem) {
+	public DataItemSerializer(final boolean useDirectMem, final boolean shouldRecordTime) {
```

Comment:
I'd consider adding doc string for shouldRecordTime parameter.

Reply:
will do